docs/content_management/rich_text/extend_online_editor.md

```diff
@@ -387,7 +387,7 @@ A CKEditor plugin is installed locally by using `yarn add ` or `npm install`, an
 A local installation looks like:
 
 ```bash
-yarn add @ckeditor/ckeditor5-special-characters
+yarn add @ckeditor/ckeditor5-special-characters@40.2.0
```

Comment:
Can we add information in this section to install plugins compatible with the current version of CKEditor? so that people don't try to install like we had before, without a version?